product/scanning.py

```diff
@@ -21,6 +22,19 @@
 from .model import ContainerImage, Component, UploadResult, UploadStatus
 
 
+class ProcessingMode(Enum):
+    UPLOAD_IF_CHANGED = 1
+    RESCAN = 2
+    FORCE_UPLOAD = 4
+
+
+class AppStatus(Enum):
```

Comment:
ditto (IntEnum / IntFlag)

also: I am unhappy with the name. I know protecode calls uploaded things "apps". But this term is much too generic for my taste. It should at least be called ProtecodeApp maybe. There must be a better name